icmd/ops.go

```diff
@@ -1,4 +1,38 @@
 package icmd
 
+import (
+	"io"
+	"time"
+)
+
 // CmdOp is an operation which modified a Cmd structure used to execute commands
 type CmdOp func(*Cmd)
+
+// WithTimeout sets the timeout duration of the command
+func WithTimeout(timeout time.Duration) func(*Cmd) {
```

Comment:
Small thing, but I think if you use `CmdOp` as the return value type it will organize these better in `godoc`. 

I think with `func(*Cmd)` it wont organize them under the type.